src/Plugin/Action/MintIdentifier.php

```diff
@@ -27,7 +29,11 @@ protected function getFieldData(): array {
     if ($data_profile) {
       foreach ($data_profile->getData() as $key => $field) {
         if ($this->entity->hasField($field)) {
-          $data[$key] = $this->entity->get($field)->getString();
+          if ($this->entity->get($field) instanceof EntityReferenceFieldItemList) {
+            $data[$key] = $this->entity->get($field)->entity->label();
+          } else {
+            $data[$key] = $this->entity->get($field)->getString();
+          }
```

Comment:
```suggestion
          $entity_field = $this->entity->get($field);
          if (!$entity_field->isEmpty()) {
            if ($this->entity->get($field) instanceof EntityReferenceFieldItemList) {
              $data[$key] = $entity_field->entity->label();
            }
            else {
              $data[$key] = $entity_field->getString();
            }
          }
```

So nit about coding standards but noticed another bug while testing this. A non-empty field existence isn't checked for at the moment 🤦.

Reply:
I took a slightly different approach on this one, but the effect is the same.